### DISCUSSION TITLE
Japanese translation

### DISCUSSION THREAD

**Discussion by maboroshin** [libre-tube/LibreTube#5022] [General]
Discussion among translators in the Japanese. Weblate's comment function requires a window to be opened for each string. It’s difficult to organize a discussion.

Thanks for improving the translations. It's correcting a few mistranslations. But in many there are conflicts.

@lapriere revert the improvement translations (C) to the intermediate translations (B). The literal translation (B) has the original English meaning. However, it is unnatural for Japanese (Improvement possible). This is where improved translations (C) come in. Of course, after brushing up, it must not contain misunderstandings. In general, software UI is devised to keep strings short (C). Generally, when you brush up on your writing, you shorten it (C). But he lengthens the sentences shortened and improved (B).

He wants to be B (or A) and I want to be C. Developer Bnyro's comments and interventions are also welcome.

I have translated over a dozen YouTube clients. @lapriere first appeared on the YouTube client [hyperion](https://github.com/zt64/Hyperion), 1week ago. He seems to revert to B with no discussion in hyperion. To me it feels like a quality decline. The same thing is happening with LibreTube. I decided to leave Hyperion as it was not important to me.

**Translation procedure?**

* (A) Transliteration -> (B) Too literal and unnatural translation : aka Direct translation like an mashine translation -> (C) Improvement / Brush up : aka Free translation / Liberal translation
* Note: Transliteration is okay as long as it is a common English word in Japan.

**Comment by maboroshin** [comment]
* "segments"
* maboroshin : scene
* lapriere : segments

The transliteration of "segments" is a technical term in Japanese ([google](https://www.google.com/search?q=%E3%82%BB%E3%82%B0%E3%83%A1%E3%83%B3%E3%83%88)). It seems to be university jargon. Instead, "scene" is maybe understood by children in Japan.

----

* "default"

I assign Kanji individually (Normal, Initial value etc). He transliterates all the same. Generally, both transliteration and kanji are used.

For example, it's Kanji because of the translation team in Windows. Many are standardized on the uncommon word "既定".

----

* "Interaction reminder" (This is a request for channel subscriptions etc)
* maboroshin : Request for the action (操作のお願い)
* lapriere : Reminder (リマインダー)

The word "Reminder" can only be imagined to time management system in Japan. ([Google](https://www.google.com/search?q=%22%E3%83%AA%E3%83%9E%E3%82%A4%E3%83%B3%E3%83%80%E3%83%BC%22))

He changed many of the names in the SponsorBlock category.

----
* "Preloading"
* maboroshin : "事前読み込み" or "先読み" (Japanese with Kanji)
* lapriere : "プリロード" (Transliteration)

In its English transliteration form, Most searches are for motorcycle parts. ([Google](https://www.google.com/search?q=%E3%83%97%E3%83%AA%E3%83%AD%E3%83%BC%E3%83%89)). If you add a search term, It's a technical term for web and programming. But it is not like a fixed translation. Lots of people can't interpret it.

If it contains Kanji, even a 15-year-old can understand it.

lapriere ignores my comment and reverts to his own translation. https://hosted.weblate.org/translate/libretube/libretube/ja/?checksum=487758e76a28600f

----

* "Accents"
* third person : accent color (I support this translation.)
* lapriere: accent

In the word "accent", in Japanese it means high or low in pronunciation.

----

* "YT Music" etc
* [YT Musicの曲] [YT Musicの動画] : maboroshin
* [YT Music の曲] [YT Music の動画] : lapriere (Added space)

They are lined up horizontally, so adding spaces makes them longer horizontally. Shorter horizontally would be better for mobile devices. What are the advantages of longer?
![image](https://github.com/libre-tube/LibreTube/assets/41102508/c3ca2f49-8be3-4122-b11d-fd4b6611eb03)

----

* "Website"
* maboroshin : Official site（公式サイト）
* lapriere : Website（ウェブサイト）

Everything here is lined with one word. So in English, making it a "website" provides uniformity of design. In Japanese, however, it can be "Official site" while maintaining uniformity in fewer characters.

**Reply by maboroshin** [reply]
> your translation of "先読み" as "pre-reading" is incorrect. 

Please provide a rationale. Search for "事前に読み込む" in [Google](https://www.google.com/search?q=%22%E4%BA%8B%E5%89%8D%E3%81%AB%E8%AA%AD%E3%81%BF%E8%BE%BC%E3%82%80%22). Then, "先読み" is also included in the search results. A common understanding is recognized among the Japanese.

**Reply by maboroshin** [reply]
Of course. "事前読み込み" was my suggestion.

**Reply by maboroshin** [reply]
@lapriere : There is no URL or logic in your opinions, is there? You haven't explained at about for natural or nuances. Suppose you claim something to be 100 points and another to be 50 points. That's all. There is no reason. I don't understand.

Logic: e.g.
* "事前読み込み" : Many Japanese can understand the meaning.
* "プリロード" : Probably not the English you learn in Japanese schools.

URL: https://github.com/libre-tube/LibreTube/discussions/5022#discussioncomment-7349535

**Reply by maboroshin** [reply]
"Website" : I have not provided an explanation for "reduced 1 character". We should talk about "Preload" first.

**Reply by maboroshin** [reply]
"Website" : I wrote: The uniformity of a few letters is maintained. Emphasis is design uniformity.

----

> the nuance is accurate.

"Preloading" : Rational explanations are lacking. Which one includes what kind of finer meaning? What is the rationale?

**Comment by maboroshin** [comment]
SponsorBlock category

Shorter is better, Because many apps insert these into other parts:  e.g. "***Preview/Recap*** is skipped", "***Preview/Recap*** 3:24". It also appears in LibreTube as a selections in a dialog.

----

* Recap in "Preview/Recap"
* maboroshin : あらすじ (Summary of story : [Google](https://www.google.com/search?q=%22%E3%81%82%E3%82%89%E3%81%99%E3%81%98%22) )
* lapriere : 振り返り (Refer to the past)

"Refer to the past" is almost always a reflection or a summary of the previous meeting. ([Google](https://www.google.com/search?q=%22%E6%8C%AF%E3%82%8A%E8%BF%94%E3%82%8A%22)) Mistranslation

----

* "Intermission/Intro Animation"
* maboroshin : 合間/導入アニメ (betweentime/Assign Kanji)
* lapriere : "休憩/イントロアニメーション" (BreakTime/Transliteration)

Looks like a mistranslation.

----

* Music in "Music: Non-Music Section"
* maboroshin : 音楽 (Kanji)
* lapriere : ミュージック (Transliteration)

As the examples show, transliteration is not common. [example sentence](https://ejje.weblio.jp/content/music#hideDictPrsWERBJ )

**Reply by maboroshin** [reply]
There is no abbreviation for "music video" in Japanese. The Transliteration-only description (ミュージック) is odd.

**Reply by maboroshin** [reply]
This would be the same if written in English. Most people can understand "music". But some people will not understand "MV".

**Reply by maboroshin** [reply]
I explained first. These category names may appear without description. (This is not true for current LibreTube, though, It is difficult to understand for those who use it without settings.) What if LibreTube notifies "skip the MV : Non-Music Section" in the future?

**Reply by maboroshin** [reply]
I said first, it's in other apps.

The message doesn't seem unnatural to me. Please give me your opinion and mainly your rationale, Not only opinions.

**Reply by maboroshin** [reply]
> I think "不要なシーンをスキップしました" would be better.

And Why is "Intro animation is best"? Please always include a reason or URL. Please put it in words. I would argue you scored 100. But. Why did you give it a score of 100?

> "アニメ"(anime) sounds like a video genre!

You mention a single word, though. "導入アニメ" is not "Anime" only.

> Please check the original meaning of Recap here

Same as above. What did you read and what logic was generated? The ***Japanese*** you used is meant to be a reflection of the past. What does reading about "English in recap" prove?

"Recap" includes examples of ***books*** and ***movies***, ***episodes*** in above dictionary. "あらすじ" is also so.
But, "振り返り" is not for such story. It's often for more cautious thinking, such as meetings and reflection.

**Comment by maboroshin** [comment]
* "Something went wrong."
* maboroshin : Unknown error (Part of the translated text.)
* lapriere : Error (Same)

There are two other errors in this app: server error and network error. Inclusion of "Something" in the original sentence would be "unknown".

**Reply by maboroshin** [reply]
* lapriele : "エラーが発生しました。" : An error has occurred. (Google translate)

You value accuracy. Where is the Japanese word for "something"? It doesn't exist.

**Reply by maboroshin** [reply]
I wrote first. I have not added. "Something" corresponds to "unknown". There are other clear error messages.

**Comment by maboroshin** [comment]
@kurages : Hi, If you have an opinion, please use your words.
![image](https://github.com/libre-tube/LibreTube/assets/41102508/ae715766-e4b8-409c-8599-ecfff7f1da64)

You are not a contributor here at GitHub. I wrote to come here on Weblate. You write that you are Japanese. So I assume you have seen Weblate's comments. 4 people have joined Weblate: ```lapriere``` ,```HiSubway``` (Same name on GitHub) ,```pklion663b96357b3a42e4```(Activity 1 year ago) ,```onippi.0830``` (3 strings only)

**Reply by kurages** [reply]
You should be aware of the following

- You should translate as faithfully as possible.
- Fidelity includes nuance.
- Machine translation is not faithful enough. It is a mistake to bring up machine translation as a basis for translation.

**Reply by maboroshin** [reply]
Of course. The idea is the same. I don't know what you are trying to say. I take issue with those things below.

* The translation is too direct and unnatural, like a machine translation. (B)
* The term is too directly translated into a technical term, Jargon. It's a difficult word for Japanese. Not for the Public. (B)

Brush up on the translation. Improve for the general public. (C) If the text contains misunderstandings, it should be improved.

**Reply by kurages** [reply]
If we were of the same mindset, we would not base our results on machine translation.

**Reply by maboroshin** [reply]
Hmm... Who used the machine translation? I'm not talking about such.

**Reply by kurages** [reply]
You.
Why don't you look back at this ~~thread~~ discussion?

**Reply by maboroshin** [reply]
Please specify which text. And what is happening in that text about machine translation.

**Reply by maboroshin** [reply]
That is for discussion here and to show transparency to English speakers. And the topics are not related.

> It is a mistake to bring up machine translation as a basis for translation.
> we would not base our results on machine translation.

The results are based on machine translation. Do you mean that the translated text was generated by machine translation? No one is talking about this topic.

**Reply by maboroshin** [reply]
You have added a different claim.

So which sentence are you referring to? Please make more specific. Too ambiguous.

**Reply by maboroshin** [reply]
There are too few explanations and I don't understand anything.

The machine translation in that #5022 is Japanese into English. #5022 would indicate that the word has disappeared. 

For example, Weblate has a machine translation feature. Many services have a translation memory feature. There will be a consensus to base such machine translation.

* Advantages of machine translation: machines do not make complete misreadings. That's the reason for the unnatural translation, though machine translation is less likely to miss a word. If a certain word is missing, it is easy for English speakers to understand.
* Translation by Human: Fundamental mistakes can be made, such as mistaking Reader and Leader.

**Reply by maboroshin** [reply]
@kurages How did you get involved here?

You are here shortly after I directed here in a comment on Weblate. Only Weblate participants will be notified. And of the four participants, all but lapriere would have been there more than a year ago. Only lapriere is currently active.

You opened the LibreTube DISCUSSIONS. Why?

**Comment by lapriere** [comment]
@maboroshin
By the way, this discussion has been labeled as "bug" but I think it should be changed to "discussion"

**Reply by maboroshin** [reply]
It wasn't my doing.

**Comment by lapriere** [comment]
## List of solved translations

:(

## List of unsolved translations

- [ ] Preloading
- [ ] segments
- [ ] default
- [ ] Accents
- [ ] "**YT Music**" related string
(Whether to add a *space "` `"* or not)
- [ ] Website
- [ ] Something went wrong.

#### SponsorBlock related
- [ ] Preview/Recap
- [ ] Interaction reminder
- [ ] "Music: Non-Music Section"

**Reply by maboroshin** [reply]
We are already talking elsewhere; the official YouTube client seems to include machine translation. It contains unnatural Japanese.

However, these strings above seem to have little to do with the official YouTube client.

**Comment by maboroshin** [comment]
@Bnyro : About @kurages

kurages does't appear to be a contributor to this project. I have no idea why he came here. He seems to be an outsider.

I'm not sure why, but here he is, And He will repeat "thumbs down". I told him to put his opinion in writing. https://github.com/libre-tube/LibreTube/discussions/5022#discussioncomment-7349441 But there seems to be little explanation. He repeats "thumbs down" instead.

There are sentences in the Code of Conduct about harassment and personal attacks. If you feel he is over the limit, Please ban him from the project.

**Reply by maboroshin** [reply]
Insulting or derogatory words like Insane would not be recommended. My point is that lapriere translates like a machine translation.

The discussion is not progressing because your opinions are not accompanied by logic or evidence.

**Reply by maboroshin** [reply]
I have already explained above.

You say I don't understand the context or the correct meaning. But you have not explained it rationally. How can English speakers agree with you?

**Comment by lapriere** [comment]
This discussion seems to be going nowhere... :(

**Comment by lapriere** [comment]
It seems that he has his own translation rules (~~whether they are generally accepted or not is another matter~~). I don't want to waste any more time, so I give up on translating this project. Good bye LibreTube :wave:

**Reply by maboroshin** [reply]
There are maboroshin rules and lapriere rules, right? Why don't we just explain the logic behind the argument? Otherwise, no one will understand the rationale.

The kurages is exactly the same. You don't rationally explain what is wrong with me. Which? Why? What sources of information can you confirm?
